### PR TITLE
v85.12.0

### PR DESCRIPTION
**Pull request by romoore**

### CODE REVIEW

**Review by fluxminds — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+## New features
+
+Support for Material Elevation expanded to two more components. APIs to support
+overriding the computed value for `mdc_baseElevation` and to respond to
+elevation changes are available. Assigning an `elevationDidChangeBlock` on a
+supported component enables code to be executed whenever the component's
+elevation changes. This may include state-based elevation changes (like
+pressing on a Chip) or when the component's elevation value is changed
+programmatically.
+
```
Comment (jverkoey) [inline CHANGELOG.md:14]: Call out the performance improvements details here as well:

```
## Performance improvements

Changes to how Ink and Ripple create colors will result in memory savings on the order of KBs for apps that have on the order of hundreds of ink/ripple instances in memory. Memory fragmentation, cache performance, malloc contention will also be reduced slightly.
```

**Review by jverkoey — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (romoore) [reply CHANGELOG.md:14]: Sorry, didn't see your review comments before I started on the merging work.

Comment (jverkoey) [reply CHANGELOG.md:14]: Please include it in the release notes on GitHub and in the internal email then.

Comment (romoore) [reply CHANGELOG.md:14]: Let's talk offline. That would violate our release process.

**Pull request merged**